### PR TITLE
Bloodlands: Grammar, Naming, and Typos

### PR DESCRIPTION
**Pull request by gnummers**
Addressing consistency in names of creatures, some grammar, and other small typo fixes.

### CODE REVIEW

File: Segments/Bloodlands.mapproj [hunk 1]
```diff
@@ -224174,7 +224174,7 @@
       <script name="OnIncomingPlayer" enabled="true">
         <block><![CDATA[]]></block>
         <block><![CDATA[
-    source.Say($"Grab yourself a recall ring, don't want to get stuck out there.");
+    source.Say($"Grab yourself a recall ring. You deon't want to get stuck out there.");
```
Comment (Chilly021) [inline Segments/Bloodlands.mapproj:224177]: spelling

File: Segments/Bloodlands.mapproj [hunk 2]
```diff
@@ -226196,17 +226196,17 @@ source.Say($"Oh, you've dealt with my pets have you, {player.Name}? let us play.
     giant.Attacks = new CreatureAttackCollection
     {
         {
-            new CreatureAttack(23,      40, 60,      "The giant smashes you with his fist.",
+            new CreatureAttack(23,      40, 60,      "The ethereal Giant smashes you with his fist.",
```
Comment (Chilly021) [inline Segments/Bloodlands.mapproj:226199]: Maybe casing on giant?

File: Segments/Bloodlands.mapproj [hunk 3]
```diff
@@ -226196,17 +226196,17 @@ source.Say($"Oh, you've dealt with my pets have you, {player.Name}? let us play.
...
                                                         new AttackProneComponent(5), 
                                                         new AttackStunComponent(5)),                40
         },
         {
-            new CreatureAttack(23,      45, 70,     "The giant stomps on you.",
+            new CreatureAttack(23,      45, 70,     "The ethereal Giant stomps on you.",
```
Comment (Chilly021) [inline Segments/Bloodlands.mapproj:226204]: Maybe casing on giant?

File: Segments/Bloodlands.mapproj [hunk 4]
```diff
@@ -226196,17 +226196,17 @@ source.Say($"Oh, you've dealt with my pets have you, {player.Name}? let us play.
...
                                                         new AttackProneComponent(10), 
                                                         new AttackStunComponent(10)),               40
         },
         {
-            new CreatureAttack(23,      55, 80,     "The giant slams you across the room.",
+            new CreatureAttack(23,      55, 80,     "The ethereal Giant slams you across the room.",
```
Comment (Chilly021) [inline Segments/Bloodlands.mapproj:226209]: Maybe casing on giant?

**Review by Chilly021 — COMMENTED** [review]
Just a few small things. One spelling error and three similar recommendations.

File: Segments/Bloodlands.mapproj (hunk 1 above)
Comment (gnummers) [reply Segments/Bloodlands.mapproj:224177]: fixed, thank you!

File: Segments/Bloodlands.mapproj (hunk 2 above)
Comment (gnummers) [reply Segments/Bloodlands.mapproj:226199]: fixed, thank you!

File: Segments/Bloodlands.mapproj (hunk 3 above)
Comment (gnummers) [reply Segments/Bloodlands.mapproj:226204]: fixed, thank you!

File: Segments/Bloodlands.mapproj (hunk 4 above)
Comment (gnummers) [reply Segments/Bloodlands.mapproj:226209]: fixed, thank you!

**Comment by gnummers** [comment]
Should be resolved now. Thanks, Chilly!

**Review by Chilly021 — APPROVED** [review]

**Review by yarrim — APPROVED** [review]

**Pull request merged**